### PR TITLE
Add travis, validate skeletons syntax

### PR DESCRIPTION
**Pull request by denar90**
It's required to switch on travis to make it works.

Here's test example
https://github.com/denar90/skeletons/pull/2
https://travis-ci.org/denar90/skeletons/builds/218604710

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - node
+script:
```
Comment (shvaikalesh) [inline .travis.yml]: this is extra: default value.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "A registry of Brunch skeletons",
   "main": "skeletons.json",
   "scripts": {
-    "test": "exit 0"
```
Comment (shvaikalesh) [inline package.json]: let's make it simpler: `node skeletons.json`

Comment (denar90) [inline package.json]: Learn Something New Every Day 👍

File: .travis.yml [hunk 3]
```diff
@@ -1,5 +1,3 @@
 language: node_js
 node_js:
-  - node
-script:
-  - npm test
\ No newline at end of file
+  - node
```
Comment (shvaikalesh) [inline .travis.yml]: I am sorry, but could you please leave trailing line break so GitHub won't show this icon?

**Comment by shvaikalesh** [comment]
Thanks 👍

**Pull request merged**